parlai/chat_service/services/messenger/message_socket.py

```diff
@@ -134,7 +134,7 @@ def run_socket(*args):
             url_base_name = self.server_url.split('https://')[1]
             while self.keep_running:
                 try:
-                    sock_addr = "ws://{}/".format(url_base_name)
+                    sock_addr = "wss://{}/".format(url_base_name)
```

Comment:
what is this change exactly?

Reply:
This change is to use WebSockets with SSL. Makes sense for all scenarios since Messenger requires SSL already for the webhooks to work.